apps/src/templates/teacherDashboard/StatsTable.jsx

```diff
@@ -138,6 +138,7 @@ class StatsTable extends Component {
       <Table.Provider
         columns={columns}
         style={styles.table}
+        id="#uitest-stats-table"
```

Comment:
nit (but also question): is the "#" needed before the id name? could it be `id="uitest-stats-table"`?